.github/workflows/build.yml

```diff
@@ -148,6 +149,11 @@ jobs:
       with:
         dotnet-version: 6.x
 
+    - name: Minio Server UP
+      uses: isbang/compose-action@v1.4.0
+      with:
+        compose-file: "./docker-compose/S3/docker-compose-minio.yml"
```

Comment:
Could you add Minio to the terraform deployment? We plan to remove docker compose from core in the (not so distant) future.

Reply:
I will do it afterwards : https://github.com/aneoconsulting/ArmoniK/issues/840